### ISSUE TITLE
delay when pressing <tab> or <s-tab>

### ISSUE THREAD

**Issue by nfwyst** [milanglacier/minuet-ai.nvim#17]
Significant jump delay when enter tab/shift-tab

this is my config snip:
```lua
local function get_onjump(step, cmp, luasnip)
  return function(fallback)
    if cmp.visible() then
      if step < 0 then
        return cmp.select_prev_item()
      end
      return cmp.select_next_item()
    end
    if luasnip.jumpable(step) then
      return luasnip.jump(step)
    end
    return fallback()
  end
end

['<tab>'] = cmp.mapping(get_onjump(1, cmp, luasnip), { 's', 'i' }),
['<s-tab>'] = cmp.mapping(get_onjump(-1, cmp, luasnip), { 's', 'i' }),
```

**Comment by milanglacier** [comment]
Have you ever tried to play with those configuration in readme?

https://github.com/milanglacier/minuet-ai.nvim?tab=readme-ov-file#significant-input-delay-when-moving-to-a-new-line

**Comment by nfwyst** [comment]
> Have you ever tried to play with those configuration in readme?
> 
> [milanglacier/minuet-ai.nvim#significant-input-delay-when-moving-to-a-new-line](https://github.com/milanglacier/minuet-ai.nvim?tab=readme-ov-file#significant-input-delay-when-moving-to-a-new-line)

Yes, I tried. but no help for this issue. it block inputing space or jumping between code snippets .

**Comment by milanglacier** [comment]
spaces are blocked by design because `minuet` uses spaces as trigger key. This is best in my experienced because LLM usually returns a complete word, it is hard to return a partial word.

however in my experience this are usually very smooth and the blockings are negligible.

Are you saying that you are using`cmp.core.view:visible()` instead of `cmp.visible` in your code snippet but it still not helps?

**Comment by nfwyst** [comment]
It did work when use `cmp.core.view:visible()`. I just use ghost text now. because its confusing when using cmp. thank you @milanglacier I really love this plugin.
